src/main/java/com/gruppe4/boredoom/backend/spring/repository/QuestionRepository.java

```diff
@@ -21,7 +25,16 @@ public QuestionRepository(@Value("${spring.datasource.url}") String url,
 
         String sqlSelectAllQuestions = "SELECT q.*, qt.question_type FROM question q JOIN question_type qt ON q.question_type_id " +
                 "= qt.id";
-        return executeSql(sqlSelectAllQuestions, this::transformQuestion);
+
+       List<Question> questionList = executeSql(sqlSelectAllQuestions, this::transformQuestion);
+
+        for (Question question : questionList) {
+            String sqlSelectChoices = "SELECT choice FROM choice c JOIN question2choice q2c ON c.id = q2c.choice_id " +
+                    "WHERE q2c.question_id = " + question.getId();
```

Comment:
`"WHERE q2c.question_id = " + question.getId();` 
Sowas ist generell anfällig für SQL-Injection. Das sollte man eigentlich als Parameter in einem Prepared Statement übergeben.
Ist jetzt in diesem Fall nicht so schlimm, weil "question.getId()" kein UserInput ist. Aber wenn du irgendwann mal Daten verarbeitest, die ein User eingibt, solltest du das nicht machen.
Sonst kann da jemand "drop database;" reinschreiben und unsere Datenbank geht kaputt :D